### PR TITLE
Simplified BeerController and added missing import

### PR DESCRIPTION
**Pull request by mraible**
## Description:

Simplified `BeerController.java` and added missing import to `beer.service.ts`.

## Type of Pull Request:

- [ ] Content (Documentation updates or typo-fixes)
- [x] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:

https://github.com/oktadeveloper/spring-boot-angular-example/pull/1

## How Has This Been Tested?
- [ ] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:
DevBlog Team

### CODE REVIEW

File: code/php/index.html [hunk 1]
```diff
@@ -132,19 +132,19 @@ <h3 class="Sidebar-title">Programming languages</h3>
 	    <!-- START Subpages Grid -->
 	<div class="Row">
 	  <div class="Column--4 Column--small-12">
-        <a href="/code/php/quickstart-signin-widget.html">
-	      <h3 class="h4">Okta Sign-In Widget</h3>
+        <a href="/code/php/simplesamlphp.html">
```
Comment (bdemers) [inline code/php/index.html]: Should these changes be in here? or just the java ones above?

File: _source/_posts/2017-04-26-bootiful-development-with-spring-boot-and-angular.md [hunk 2]
```diff
@@ -167,16 +167,11 @@ public class BeerController {
...
 
         return repository.findAll().stream()
                 .filter(this::isGreat)
-                .map(b -> {
-                    Map<String, String> m = new HashMap<>();
-                    m.put("id", b.getId().toString());
-                    m.put("name", b.getName());
-                    return m;
-                }).collect(Collectors.toList());
+                .collect(Collectors.toList());
```
Comment (bdemers) [inline _source/_posts/2017-04-26-bootiful-development-with-spring-boot-and-angular.md]: Much better!

**Comment by mraible** [comment]
I didn't make those changes. I'm guessing someone forgot to run the prod build on master, so this picked up those changes.

File: code/php/index.html (hunk 1 above)
Comment (rchild-okta) [inline code/php/index.html]: FYI there seems to be a difference between what jekyll is outputting depending on the machine - it's a no-op change (switches the order of the menu items), but it is strange that on some computers it outputs in this order and in others a different order. Can probably ignore for now while we fix it.

**Comment by rchild-okta** [comment]
FYI @mraible it's fixed in latest master with this commit: https://github.com/okta/okta.github.io/pull/850

**Comment by bdemers** [comment]
Rebase?

**Comment by mraible** [comment]
Closing. Will create a new one.

**Pull request closed without merging**